filters.go

```diff
@@ -194,3 +194,16 @@ func (p *Pipe) Freq() *Pipe {
 	}
 	return Echo(output.String())
 }
+
+// Column reads from the pipe, and returns a new pipe containing only the Nth
```

Comment:
Fair point, but I think this is OK—it's what `cut` does, and that's probably because we think intuitively of the 'first' column not the 'zeroth' column.

Reply:
I thought the unix tools are not the design driver... 😉

Reply:
No, indeed—and where the Unix API departs from common sense, I won't slavishly follow it. On the other hand, where it _is_ sensible, I won't depart from it just for the sake of being different!